docs/source/guide.rst

```diff
@@ -29,7 +29,7 @@ The hook will retrieve a balance of an ERC20 token of the provided address.
 
 .. code-block:: javascript
 
-  function useTokenBalance(address: string | Falsy, tokenAddress: string | Falsy) {
+  function useTokenBalance(tokenAddress: string | Falsy, address: string | Falsy) {
```

Comment:
Shouldn't this change be in https://github.com/EthWorks/useDApp/pull/85 ?

Reply:
Sure, I forgot about this file when I was making the changes.